app/src/main/java/com/framgia/soundclound/screen/home/HomeViewModel.java

```diff
@@ -1,21 +1,27 @@
 package com.framgia.soundclound.screen.home;
 
+import android.content.Context;
+import android.content.Intent;
 import android.databinding.BaseObservable;
 import android.databinding.Bindable;
-
 import com.framgia.soundclound.BR;
 import com.framgia.soundclound.data.model.Genre;
 import com.framgia.soundclound.data.source.repository.GenresRepository;
+import com.framgia.soundclound.screen.detailgenre.GenreDetailActivity;
+import com.framgia.soundclound.util.Constant;
```

Comment:
ERROR: Unused import: com.framgia.soundclound.util.Constant. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

Reply:
remove unsed import